modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js

```diff
@@ -32,7 +32,13 @@ function getSessionClickFormData(cmd) {
 }
 
 function getSessionClickURL() {
-	return `${Liferay.ThemeDisplay.getPortalURL()}${Liferay.ThemeDisplay.getPathMain()}/portal/session_click`;
+	let sessionClickURL = `${Liferay.ThemeDisplay.getPortalURL()}${Liferay.ThemeDisplay.getPathMain()}/portal/session_click`;
+	
+	if (Liferay.ThemeDisplay.getScopeGroupId()) {
```

Comment:
As per the way we generate `Liferay.ThemeDisplay` object, the call to this function would never return falsy result. This is consistent what with we're assuming in the tests, where we always expect the `doAsGroupId` parameter. I'm fine with this check though

Reply:
Actually, this seems to be an issue in final review by Brian:

> _We should only set that param if we explicitly set doAsGroupId_

Although I'm not 100% sure how to check we explicitly set that param. Would be enough to check the scopeGroupId > 0?